src/concrete_convex_hull.jl

```diff
@@ -28,8 +28,8 @@ The convex hull as a list of vectors with the coordinates of the points.
 
 ### Algorithm
 
-A pre-processing step treats the cases with `0`, `1` and `2` points in any dimension.
-For more than `3` points, the algorithm used depends on the dimension.
+A pre-processing step treats the cases with `0`, `1`, `2` and `3` points in any dimension.
+For more than `4` points, the algorithm used depends on the dimension.
```

Comment:
This was ambiguous before, but we can take the chance and improve it now.
```suggestion
For `4` or more points, the algorithm used depends on the dimension.
```